src/main/java/me/jellysquid/mods/sodium/client/gui/options/OptionImpact.java

```diff
@@ -1,13 +1,14 @@
 package me.jellysquid.mods.sodium.client.gui.options;
 
+import net.minecraft.client.resource.language.I18n;
 import net.minecraft.util.Formatting;
 
 public enum OptionImpact {
-    LOW(Formatting.GREEN, "Low"),
-    MEDIUM(Formatting.YELLOW, "Medium"),
-    HIGH(Formatting.GOLD, "High"),
-    EXTREME(Formatting.RED, "Extreme"),
-    VARIES(Formatting.WHITE, "Varies");
+    LOW(Formatting.GREEN, I18n.translate("sodium.option_impact.low")),
```

Comment:
Will this affect in-game language changing? (Using translated strings in enum constructor)